source/opt/common_uniform_elim_pass.h

```diff
@@ -190,20 +189,19 @@ class CommonUniformElimPass : public Pass {
   Pass::Status ProcessImpl();
 
   // Map from uniform variable id to its common load id
-  std::unordered_map<uint32_t, uint32_t> uniform2load_id_;
+  CAUnorderedMap<uint32_t, uint32_t> uniform2load_id_;
 
   // Map of extract composite ids to map of indices to insts
   // TODO(greg-lunarg): Consider std::vector.
-  std::unordered_map<uint32_t,
-                     std::unordered_map<uint32_t, std::list<Instruction*>>>
+  CAUnorderedMap<uint32_t, CAUnorderedMap<uint32_t, std::list<Instruction*>>>
```

Comment:
forgot the std::list. It's also a node-base container that will probably benefit from using an allocator.